support-frontend/assets/components/directDebit/directDebitForm/directDebitForm.tsx

```diff
@@ -1,258 +1,140 @@
 		<div>
-			<AccountHolderNameInput
-				onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-					dispatch(setAccountHolderName(e.target.value))
-				}
-				value={accountHolderName}
+			{/** BACS requirement:
+        "Name of the account holder, as known by the bank. Usually this is the
+        same as the name stored with the linked creditor. This field will be
+        transliterated, upcased and truncated to 18 characters."
+        https://developer.gocardless.com/api-reference/
+      * */}
+			<TextInput
+				label="Account name"
+				id="accountHolderName"
+				data-qm-masking="blocklist"
+				value={props.accountHolderName}
+				onChange={(e) => props.updateAccountHolderName(e.target.value)}
+				maxLength={40}
+				error={props.errors.accountHolderName?.[0]}
 			/>
 
 			<div css={accountNumberSortCodeContainer}>
-				<SortCodeInput
-					onChange={(e) => dispatch(setSortCodeString(e.target.value))}
-					sortCodeString={sortCodeString}
-				/>
+				<div>
+					<TextInput
+						label="Sort code"
+						id="sortCodeString"
+						data-qm-masking="blocklist"
+						value={props.sortCode}
+						onChange={(e) => props.updateSortCode(e.target.value)}
+						pattern="[0-9]*"
+						minLength={6}
+						maxLength={6}
+						inputMode="numeric"
+						error={props.errors.sortCodeString?.[0]}
+					/>
+				</div>
 
-				<AccountNumberInput
-					onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-						dispatch(setAccountNumber(e.target.value))
-					}
-					value={accountNumber}
-				/>
+				<div>
+					<TextInput
+						label="Account number"
+						id="accountNumber"
+						data-qm-masking="blocklist"
+						value={props.accountNumber}
+						onChange={(e) => props.updateAccountNumber(e.target.value)}
+						pattern="[0-9]*"
+						minLength={6}
+						maxLength={10}
+						inputMode="numeric"
+						error={props.errors.accountNumber?.[0]}
+					/>
+				</div>
 			</div>
 
-			<ConfirmationInput
-				onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-					dispatch(setAccountHolderConfirmation(e.target.checked))
+			<Checkbox
+				id="accountHolderConfirmation"
+				label="I confirm that I am the account holder and I am solely able to
+							authorise debit from the account"
+				error={!!props.errors.accountHolderConfirmation?.[0]}
+				onChange={(e) =>
+					props.updateAccountHolderConfirmation(e.target.checked)
 				}
-				checked={accountHolderConfirmation}
+				checked={props.accountHolderConfirmation}
 			/>
 
-			<RecaptchaInput
-				setRecaptchaToken={(token) => dispatch(setRecaptchaToken(token))}
-				expireRecaptchaToken={() => dispatch(expireRecaptchaToken())}
-			/>
+			{props.recaptcha && (
+				<div css={recaptcha}>
+					<ElementDecorator
+						id="robot-checkbox"
+						text="Security check"
+						error={props.errors.recaptcha?.[0]}
+						renderElement={() => props.recaptcha}
+					/>
+				</div>
+			)}
 
-			{formError && (
+			{props.formError && (
 				<div
 					css={css`
 						margin-top: 8px;
```

Comment:
could be:

`margin-top: ${space[2]}px;`

Reply:
This wasn't added as part of this PR so I think I'll leave this alone for now.